account_budget_activity/models/account_budget.py

```diff
@@ -127,8 +127,9 @@ class AccountBudget(models.Model):
     # )
     released_amount = fields.Float(
         string='Released Amount',
-        compute='_compute_amount',
+#         compute='_compute_amount',
```

Comment:
If ths method _compute_amount is not used, please remove it. (this apply to other part of code)